dbus/methods.go

```diff
@@ -839,3 +839,38 @@ func (c *Conn) FreezeUnit(ctx context.Context, unit string) error {
 func (c *Conn) ThawUnit(ctx context.Context, unit string) error {
 	return c.sysobj.CallWithContext(ctx, "org.freedesktop.systemd1.Manager.ThawUnit", 0, unit).Store()
 }
+
+type Process struct {
+	Path    string // Where this process exists in the unit/cgroup hierarchy
+	PID     uint64 // The numeric process ID (PID)
+	Command string // The process command and arguments as a string
+}
+
+// GetUnitProcessesContext returns an array with all currently running processes in a unit *including* its child units.
+func (c *Conn) GetUnitProcessesContext(ctx context.Context, unit string) ([]Process, error) {
```

Comment:
I think we can directly call this `GetUnitProcesses`, plus avoid the separate `*Internal` helper.